engines/router/missionctl/handlers/batch_http_handler.go

```diff
@@ -0,0 +1,124 @@
+package handlers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"sync"
+
+	"github.com/gojek/turing/engines/router/missionctl"
+	"github.com/gojek/turing/engines/router/missionctl/errors"
+	"github.com/gojek/turing/engines/router/missionctl/log"
+	"github.com/gojek/turing/engines/router/missionctl/turingctx"
+)
+
+type batchHTTPHandler struct {
+	httpHandler
+}
+
+type batchResponse struct {
+	StatusCode int         `json:"code"`
+	ErrorMsg   string      `json:"error,omitempty"`
+	Data       interface{} `json:"data,omitempty"`
+}
+
+type batchResult struct {
+	BatchResult []batchResponse `json:"batch_result"`
+}
```

Comment:
Also actually is StatusCode really required? Can't it be taken from the HTTP header?

Reply:
When i checked, it was not available from the route response, there was only application/content in the header. 

Also, these are for the individual resquest's, I cant think of any other way to depict them in case of partial failures.

Reply:
Not sure what you mean it's here: https://github.com/gojek/turing/blob/5cdd2094c69ffeab327605d513eede288c5f4247/engines/router/missionctl/handlers/batch_http_handler.go#L97

Reply:
yup that is for the header/status for the batch request, the status code is meant for individual responses from the route.

![image](https://user-images.githubusercontent.com/30390872/142795023-dc568c94-3837-4857-a670-5e42c8c7a294.png)

Reply:
no thats the payload not the http headers.